api/tests.py

```diff
@@ -71,3 +80,274 @@ def test_options_manifest_data(self):
+            }
+        ]
+    }
+        """
+
+        i = Importer(ImportOptions(update=True, append=False, testrun=False))
+        i.import_document(json.loads(self.test_document_json),ImportSpec("test_filename","test_model_class","import_monster"))
+        i.import_class(
+            json.loads(self.test_charclass_json,strict=False),
+            ImportSpec(
+                "test_filename",
+                "test_model_class",
+                "import_class",
+                sub_spec=ImportSpec(
+                    "test_filename", 
+                    Archetype,
+                    i.import_archetype
+                    )
+                )
+            )
+
+    def test_get_classes(self):
+        response = self.client.get(f'/classes/?format=json')
+
+        # Confirm the basic elements show up.
+        self.assertContains(response,'count',count=4)
+        self.assertContains(response,'next',count=3)
+        self.assertContains(response,'previous',count=1)
+        self.assertContains(response,'results',count=1) 
+
+    def test_get_class_data(self):
+        response = self.client.get(f'/classes/?format=json')
+
+        in_class =  json.loads(self.test_charclass_json, strict=False)
+        out_class = response.json()['results'][0]
+
+        self.assertEqual(in_class['name'], out_class['name'])
+        self.assertEqual(in_class['features']['desc'], out_class['desc']) #The Features key is not duplicated in the api
+        self.assertEqual(in_class['features']['hit-dice'], out_class['hit_dice']) # dashes on entry, underscores out.
```

Comment:
If you wind up adding the bulk-assertions pattern that I recommended for Monsters, you could handle this too:

```py
equal_fields = ('hp-at-1st-level', 'prof-armor', ...)
for input_field_name in equal_fields:
    # Input JSON uses hyphens; output JSON uses underscores.
    output_field_name = field_name.replace('-', '_')
    self.assertEqual(in_class[input_field_name], out_class[output_field_name])
```